test/CONVERT_TO_IDIOMATIC.md

```diff
@@ -0,0 +1,60 @@
+# Converting our test suite to idiomatic Clojure tests
+
+## Goals
+
+- Make the test suite more familiar to other Clojure programmers by following
```

Comment:
This is the minorest of minor comments, but, if these goals are supposed to be in order, i would suggest putting this one last. It's important, but I think the other goals listed here are more important.